test/dynamo/test_higher_order_ops.py

```diff
@@ -3461,28 +3635,6 @@ def wrapper_fn(x):
             )
             self.assertEqual(actual, expected)
 
-    @config.patch(capture_func_transforms=True)
-    def test_vmap_illegal_op_graph_break(self):
-        counters.clear()
-
-        def bad_fn(x):
-            x.stride()
-            return x
-
-        def wrapper_fn(x):
-            return torch.func.vmap(bad_fn)(x)
-
-        x = torch.randn(3, 3, 3)
-        actual = wrapper_fn(x)
-        expected = torch.compile(wrapper_fn, backend="aot_eager", fullgraph=False)(x)
-        self.assertEqual(len(counters["graph_break"]), 1)
-        assert_dict_matches_regex(
-            self,
-            dict(counters["graph_break"]),
-            {".*Illegal getattr invocation stride in strict mode": 2},
-        )
-        self.assertEqual(actual, expected)
-
```

Comment:
Can we keep this test, but instead try to return the stride in a situation where vmap changes the strides?

vmap over F.convolution sometimes changes the strides

Reply:
Should the following fail on dynamo?

```python
def bad_fn(x):
    return x.view((4, 3))

def wrapper_fn(x):
    return torch.func.vmap(bad_fn)(x)

x = torch.randn(2, 3, 4)
actual = wrapper_fn(x)
expected = torch.compile(wrapper_fn, backend="aot_eager", fullgraph=False)(x)
```

Reply:
If it passes in eager mode then it should pass with Dynamo

Reply:
Just saw the comment you made a while ago on @kshitij12345 PR
https://github.com/pytorch/pytorch/pull/101707#discussion_r1278030605

> We need to make sure .stride() and .storage_offset() (are there more?) are never accessed in this speculate_subgraph. Otherwise, the captured graph might be wrong, because vmap has different behavior w.r.t. strides and storage offset than regular eager-mode pytorch.

With the new vmap implementation, calling `.stride()` or `.storage_offset()` can still produce an invalid graph? I couldn't find a test case that replicates this.